src/lib/OneClickDownloadButton.svelte

```diff
@@ -16,16 +16,20 @@
 
 {#if mapId}
   <a
-    title="OneClick Download via BeatSaver and ModAssistant"
+    title="OneClick&trade; Install via BeatSaver and ModAssistant"
     href="beatsaver://{mapId}"
-    class="one-click-download-link"><Fa icon={faCloudDownloadAlt} /></a
+    class="one-click-download-link"
   >
+    <Fa icon={faCloudDownloadAlt} />
+  </a>
 {:else if playlistUrl}
   <a
-    title="OneClick Download via BeatSaver and ModAssistant"
+    title="OneClick&trade; Install via BeatSaver and ModAssistant"
```

Comment:
It _is_ listed in ModAssistant with the TM as well as all over the BSMG Wiki too who maintains ModAssistant and the OneClick option.

Reply:
I guess it's a joke in some way then. :shrug: I've seen it in ModAssistant as well. Otherwise, I'd be doubting the validity of the trade mark :smile: